### PR TITLE
detox-green-tea

### PR DESCRIPTION
**Pull request by aindrila2412**

### CODE REVIEW

**Comment by aindrila2412** [comment]
I request a maintainer to accept my PR by adding "hacktoberfest-accepted" label to it.

File: detox-green-tea.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {
+            "quantity": "1",
+            "measure": "slice",
+            "ingredient": "Lemon"
+        },
+        {
+            "quantity": "1",
+            "measure": "ltr",
+            "ingredient": "Water"
+        },
```
Comment (alfg) [inline detox-green-tea.json:35]:
```suggestion
        }
```

File: detox-green-tea.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "image": "detox-green-tea.jpg",
+    "keywords": [
+        "green tea",
+        "mint leaves",
+        "lemon"
+        "slice"
+        "water"
+        "basil"
+        "mint"
+        "leaves"
```
Comment (alfg) [inline detox-green-tea.json:55]: Missing commas for valid json.

File: detox-green-tea.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            "ingredient": "Water"
+        },
+    ],
+    "directions": [
+        "Heat a cup of water in a pan.",
+        "As water starts boiling off the flame, and put green tea leaves and dried basil.",
+        "Cover it for 1 min and sevie it."
+        "Cool it down. Meanwhile add mint leaves, lemon slice and honey in a jar."
+        "Now add this cooled tea."
+        "Add remaining water in jar, and mix well."
```
Comment (alfg) [inline detox-green-tea.json:43]: Missing commas for valid json.

**Review by alfg — CHANGES_REQUESTED** [review]
Hi, added a few suggestions to fix valid json. Thanks!

File: detox-green-tea.json [hunk 4]
```diff
@@ -6,53 +6,53 @@
...
-        "honey"
+        "lemon",
+        "slice",
+        "water",
+        "basil",
+        "mint",
+        "leaves",
+        "honey",
     ]
 }
```
Comment (aindrila2412) [inline detox-green-tea.json]: Hey, I have resolved the issue.
Can you please check it and approve my PR with Hacktoberfest label

**Comment by alfg** [comment]
Hi @aindrila2412, 

Thanks for the update. However, the JSON still seems to be invalid. Try running it through a validator, such as https://json.pizza to highlight the issues.

**Review by alfg — CHANGES_REQUESTED** [review]
See recent comment. Thanks.

**Review by alfg — CHANGES_REQUESTED** [review]
The recipe JSON goes into `src/recipes/` and images go into `src/assets/recipes/` folders respectively. Thanks.

**Pull request closed without merging**